.github/workflows/test.yml

```diff
@@ -32,3 +32,44 @@ jobs:
     - run: python -m pip install .
     - run: python -m pip install pytest
     - run: python -m pytest
+
+  test-aarch64:
+    name: "Test ${{ matrix.pyver }} aarch64"
+    strategy:
+      matrix:
+        pyver: [cp39-cp39]
+        arch: [aarch64]
+      fail-fast: false
+    runs-on: ubuntu-latest
+    env:
+      py: /opt/python/${{ matrix.pyver }}/bin/python
+      img: quay.io/pypa/manylinux2014_${{ matrix.arch }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1
+    - name: Install tools
+      run: |
+        docker run --rm -v ${{ github.workspace }}:/ws:rw --workdir=/ws \
+          ${{ env.img }} ${{ env.py }} -m pip install -U  pip build
```

Comment:
What is this step for? You are not using `build` later, and `venv` installs `pip` from the built-in copy (which means you need to up date it after installing, but you do that correctly below). Can you just remove this step?

Reply:
```suggestion
```